test/crd.go

```diff
@@ -260,3 +263,34 @@ func ManualService(svc *v1alpha1.Service) *v1alpha1.Service {
 // environment without conflicting. This method will seed rand with the current time when
 // called for the first time.
 var AppendRandomString = helpers.AppendRandomString
+
+// ServiceNameForTest generates a random service name based on the test name.
+func ServiceNameForTest(t *testing.T) string {
+	return AppendRandomString(makeK8sNamePrefix(strings.TrimPrefix(t.Name(), "Test")))
```

Comment:
make "Test", a const, given it's reused.

Reply:
Done